tests/functional/test_install_config.py

```diff
@@ -24,10 +24,10 @@ def test_command_line_options_override_env_vars(script, virtualenv):
     Test that command line options override environmental variables.
 
     """
-    script.environ['PIP_INDEX_URL'] = 'https://b.pypi.python.org/simple/'
+    script.environ['PIP_INDEX_URL'] = 'https://www.pypi.org/simple/'
```

Comment:
What's going on here? You're changing a reference to an invalid URL (AFAIK, https://b.pypi.python.org/simple/ isn't a real place) to a valid one. That seems like it's going to change the test...

Reply:
Ah right, I thought we still had the ``N.pypi.python.org`` aliases in place for legacy PyPI, forgot we removed them at some point.

Reply:
I'm going to update this test to use ``https://example.com/simple/`` to make it clear this is explicitly an invalid URL.

Reply:
That sounds like a good idea. Although I wonder if the original intent of the test was to have a *valid* URL, if that b.pypi URL was valid once. Who knows, at this stage?

Reply:
I suspect the intent was a valid URL, but that the way the test is written it doesn't matter. The test doesn't care if something gets installed or not, and it's just asserting against the debug output which will be there regardless.